CMakeLists.txt

```diff
@@ -396,6 +397,15 @@ if (LLAMA_LLAMAFILE)
     set(GGML_SOURCES_LLAMAFILE sgemm.cpp)
 endif()
 
+if (LLAMA_AARCH64)
+    set(GGML_HEADERS_AARCH64 ggml-aarch64.h)
+    set(GGML_SOURCES_AARCH64 ggml-aarch64.c)
+endif()
+
+if (LLAMA_QKK_64)
+    add_compile_definitions(GGML_QKK_64)
+endif()
```

Comment:
`LLAMA_QKK_64` has been removed

Reply:
We removed `LLAMA_QKK_64` in the latest commit.